jwst/skymatch/tests/test_skymatch.py

```diff
@@ -162,21 +162,24 @@ def _add_bad_pixels(im, sat_val, dont_use_val):
     'skymethod, subtract, skystat, match_down, grouped',
     tuple(
         product(
-            ['local', 'match', 'global', 'global+match'],
+            ['local', 'match', 'global', 'global+match', 'user'],
```

Comment:
I would like to suggest moving `'user'` test to its own separate test. None of the `subtract`, `skystat`, `match_down` apply for `skymethod='user'` and so this "user" test is run 4x2x2 times needlessly.

Reply:
Thanks for this good suggestion, please let me know if the most recent commit resolves this